index.html

```diff
@@ -1,11 +1,132 @@
+          <p class="content">Lorem ipsum dolor sit amet, consectetur adipisicing elit,
+               ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip.</p>
+              <div class="container__btn">
+                <button class="btn"><span class="read-comment fa fa-1x fa-comment-o">Read Comments</span></button>
+                <button class="btn"><span class="read-comment fa fa-1x fa-cloud-upload">Read Later</span></button>
+                <button class="btn__selected"><span class="read-comment fa fa-1x fa-share-square-o">Share This Post</span></button>
+              </div>
+        </div>
+      </div>
+      <div class="container__sidebar">
+        <div class="sidebar">
+          <img src="img/Shaun-Willis.jpg" alt="" class="author-photo">
+          <h6 class="author-name">Shaun Willis</h6>
+          <ul class="social__media-list">
+            <li class="social__media-item">Twitter</li>
+            <li class="social__media-item">Facebook</li>
+            <li class="social__media-item">Instagram</li>
+          </ul>
+          <div class="image__sidebar">
+            <img src="img/paddling.jpeg" alt="" class="image__sidebar-item">
+            <a href="#" class="ad-link">About this ad.</a>
+            <div class="container__sidebar-content">
+              <div class="sidebar-content">
+                <h6 class="sidebar-content__title">Paddling Lessons</h6>
+                <p class="sidebar-content__plaintext">Lorem ipsum dolor sit
+                   amet, consectetur adipisicing elit, sed do eiusmod tempor
+                   incididunt ut labore et dolore magna aliqua. Ut enim ad minim
+                    veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip </p>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+    </section>
+  </div>
+
+
+  <section class="container__section-footer">
+    <div class="section-footer">
+      <div class="container__footer-title">
+        <h3 class="footer-title">MOST POPULAR POST</h3>
+      </div>
+      <div class="container__footer-img">
+        <div class="footer-img">
+        <img src="img/paddling-2.jpeg" alt="" class="footer-image">
+            <h6 class="img-header img--1">ANOTHER COOL POST</h6>
+          </div>
+        <div class="footer-img">
+          <img src="img/paddling-3.jpeg" alt="" class="footer-image">
+        <h6 class="img-header">POST TITLE</h6>
+      </div>
+        <div class="footer-img">
+          <img src="img/surfing-3.jpeg" alt="" class="footer-image">
+        <h6 class="img-header">HERE'S ANOTHER LONG POST TITLE</h6>
+      </div>
+        <div class="footer-img">
+          <img src="img/surfing-4.jpeg" alt="" class="footer-image">
+        <h6 class="img-header">OH COOL A POST</h6>
+      </div>
+      </div>
+        <div class="logo">
+          <div class="container__logo">
+        <img src="img/the-iron-yard-logo.png" alt="" class="logo-pic">
```

Comment:
Be sure to close the `container__logo` before the nav list and then you can remove the width for `nav__list` which can help in getting the bottom nav to line up with the rest of the content on the right edge.